install_files/ansible-base/group_vars/all/securedrop

```diff
@@ -40,5 +40,5 @@ securedrop_cond_reboot_file: /tmp/sd-reboot-now
 
 # If you bump this, also remember to bump in molecule/builder/tests/vars.yml
 securedrop_pkg_grsec:
-  ver: "4.14.154"
+  ver: "4.14.175"
   depends: "linux-image-4.14.154-grsec-securedrop,linux-image-4.4.182-grsec,linux-firmware-image-4.4.182-grsec,intel-microcode"
```

Comment:
The `depends:` line should be updated with the new version package. Overlooked that in my first round of changes. Will amend, build, update https://github.com/freedomofpress/securedrop-dev-packages-lfs/pull/37, then add the test changes suggested by @emkll.

Reply:
Done. Ready for re-review.